web/src/utils/date.ts

```diff
@@ -29,13 +29,17 @@ export const getCurrentYear = () => {
 }
 
 export const parseTime = (millisecond: number) => {
```

Comment:
it seems have problem this function, try test with '5500000'

Reply:
OK